app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "akshat@vinsol.com"
```

Comment:
extract email in constants.. So it can be changed anytime, and changes at all places.